tools/building.py

```diff
@@ -204,77 +192,26 @@ def make_paths_absolute(f):
 # Runs llvm-nm for the given list of files.
 # The results are populated in nm_cache
 def llvm_nm_multiple(files):
+  global nm_cache
+
   with ToolchainProfiler.profile_block('llvm_nm_multiple'):
     if len(files) == 0:
       return []
     # Run llvm-nm on files that we haven't cached yet
     llvm_nm_files = [f for f in files if f not in nm_cache]
 
-    # We can issue multiple files in a single llvm-nm calls, but only if those
-    # files are all .o or .bc files. Because of llvm-nm output format, we cannot
-    # llvm-nm multiple .a files in one call, but those must be individually checked.
-
-    o_files = [f for f in llvm_nm_files if os.path.splitext(f)[1].lower() in ['.o', '.obj', '.bc']]
-    a_files = [f for f in llvm_nm_files if f not in o_files]
-
-    # Issue parallel calls for .a files
-    if len(a_files) > 0:
-      results = shared.run_multiple_processes([[LLVM_NM, a] for a in a_files], pipe_stdout=True, check=False)
```

Comment:
how slower is it without multiple processes?

Reply:
An order of magnitude on Windows at least.

Reply:
That is interesting.. so we are going from a parallel process to completely sequential one we get a big win?    I guess we should have been doing this way from the very beginning.  

Exciting to see that the last remaining use of parallelism in the python toolchain itself is now just building system libraries (I think)?